### DISCUSSION TITLE
Couldn't use withoutApproval() showing this error

### DISCUSSION THREAD

**Discussion by khagesh007** [cjmellor/approval#21] [Q&A] [answered]
`Non-static method App\\Models\\Property::withoutApproval()`

I am trying to use it like this 

`Property::withoutApproval()->create($validatedData);`

**Comment by cjmellor** [comment, marked as answer]
Hi,

The `withoutApproval()` method cannot be called statically.

```php
$property = new Property();

$property->withoutApproval()->create(...);
```

Will work

**Reply by khagesh007** [reply]
Thank you. it worked.
